res/css/views/rooms/_EventBubbleTile.scss

```
@@ -126,7 +126,9 @@ limitations under the License.
         margin: 0 -12px 0 -9px;
         > a {
             position: absolute;
-            left: -48px;
+            padding: 10px 20px;
+            top: -0px;
```

this is a bit funky, does `-0px` behave differently to `0` somewhere?

---

It does not, it's actually something that I'd like enforced by the linter. Will remove now 

```suggestion
            top: 0;
```